seq2seq/third_party/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Google LLC
```

Comment:
This looks like this file
https://github.com/huggingface/transformers/blob/v3.5.1/examples/seq2seq/__init__.py
So unless you wrote it independently, we should mark it as copied from HF and treat it as a file in category 2 file from the doc.